traffic_ops/traffic_ops_golang/deliveryservice/request/requests.go

```diff
@@ -383,6 +401,20 @@ func (req *deliveryServiceRequestAssignment) Update() (error, error, int) {
 	req.DeliveryServiceRequestNullable = current.DeliveryServiceRequestNullable
 	req.AssigneeID = assigneeID
 
+	err, existingLastUpdated := CheckIfExistsBeforeUpdate(req.ReqInfo.Tx, *req.ID)
```

Comment:
This whole block seems to be identical in a lot of places. Could it be abstracted into something like `func IsUnmodified(tx *sql.Tx, id int, tableName string, hdr http.Header) bool` ?